app/src/main/java/com/sopt/now/SignupActivity.kt

```diff
@@ -0,0 +1,59 @@
+package com.sopt.now
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.sopt.now.databinding.ActivitySignupBinding
+
+class SignupActivity : AppCompatActivity() {
+    private lateinit var binding : ActivitySignupBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignupBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        // Toast 출력될 문자열
+        var message : String?
+
+        // 회원가입 정보 저장
+        val id = binding.etSignupId.text
+        val pw = binding.etSignupPw.text
+        val nickname = binding.etSignupNickname.text
+        val mbti = binding.etSignupMbti.text
+
+        // 회원가입 버튼 클릭이벤트
+        binding.btSignupSignup.setOnClickListener {
+            // 입력 조건 + 공백 문자만 있지 않은지 확인
+            when {
+                id.isBlank() || id.length < 6 || id.length > 10 ->
```

Comment:
```suggestion
        // 회원가입 버튼 클릭이벤트
        binding.btSignupSignup.setOnClickListener {
            // 입력 조건 + 공백 문자만 있지 않은지 확인
            when {
                id.isBlank() || id.length !in 6..10->
```

이런식으로 kotlin 문법을 활용 할 수 있을 것 같아요~

Reply:
코틀린 문법 사용하는게 훨씬 깔끔하고 좋네요!! 수정해보겠습니다 감사해요!